config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
```

Comment:
I think you don't have upload checkstyle.

Reply:
I need this file to run checkstyle in my project friend.

Reply:
I think is not necessarily upload this file because this file create every time that you execute gradle check, for this reason this file not is necessary to upload to git repositories.

Reply:
Thank you, this wrong was fixed.